### DISCUSSION TITLE
Custom Tab Name Question

### DISCUSSION THREAD

**Discussion by mdurrani808** [executablebooks/meta#555] [Q&A]
Hi all,

Let's say I want to create 3 code tabs: one for Java, one for C++ (Header), and one for C++ (Source). How does one go about that in sphinx-design? From what I'm reading, the names for code tabs are automatically generated from the language type. Thank you in advance!

**Comment by thephez** [comment]
I've needed to do this too and been unable to figure it out. Related issue I opened: https://github.com/executablebooks/sphinx-design/issues/154.

**Reply by agoose77** [reply]
I gave a short reply here :) https://github.com/executablebooks/sphinx-design/issues/154#issuecomment-1930282325

**Comment by thephez** [comment]
The solution can be found in this comment and its example: https://github.com/executablebooks/sphinx-design/issues/154#issuecomment-1930655886.
